packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -2257,6 +2259,11 @@ class ComboBoxInternal extends React.Component<IComboBoxInternalProps, IComboBox
           return;
         }
 
+        // eslint-disable-next-line deprecation/deprecation
+        if (allowParentArrowNavigation && (ev.keyCode === KeyCodes.left || ev.keyCode === KeyCodes.right)) {
+          return;
+        }
```

Comment:
it might make sense to move this out of default and into a
```js
case KeyCodes.left:
case KeyCodes.right:
   if (allowParentArrowNavigation) {
     return;
  }
```

block, just for readability.

Reply:
I considered that but decided to add it to the `default` block as it does the best to preserve existing behavior as we really want to switch on `allowParentArrowNavigation`. We'd have to do a fall through from the left/right switch which seems like asking for bugs :)